src/util.ts

```diff
@@ -76,8 +76,7 @@ export function assertDefined<T>(a: T | undefined): T {
 }
 
 export async function mapDefinedAsync<T, U>(
-    arr: Iterable<T>, mapper: (t: T) => Promise<U | undefined>):
-Promise<Array<awaited U>> {
+    arr: Iterable<T>, mapper: (t: T) => Promise<U | undefined>): Promise<U[]> {
```

Comment:
Running `npm install` gave me the latest `next`, rather than the one recorded in the lockfile, and it seems like `awaited` was recently reverted.